src/utils/errors.rs

```diff
@@ -93,7 +98,18 @@ impl ApiError {
         }
     }
 }
+impl OpenApiResponderInner for ApiError {
```

Comment:
What is this for?

Reply:
I just checked and this trait needs to be implemented for any type returned on the route level.

This does that for the error (i still need to check how is it generating for any `T` in a successful case).